agent/getconf.py

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/env python3
```

Comment:
It is being used by bench-scripts as well so I though of keeping it in a common place.

Reply:
The agent (bench-scripts as well as tool-scripts) has utils-scripts in its PATH, so it can find it in util-scripts (In fact, the util-scripts *are* the "common" scripts, so puttting it there is apprripriate).